### ISSUE TITLE
Accessibility issue v12.2.0 - Product Sync/Getting Started landing pages buttons have contrast issue between foreground and background colours

### ISSUE THREAD

**Issue by Alex-Lovell** [umbraco/Umbraco-CMS#15067]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.6.0 or v12.2.0

### Bug summary

When I run a lighthouse report on the Umbraco landing page, it flags that the 'Product Sync' button on the Product Sync tab has a contrast issue. The same applies to the 'Get Started' page that appears in a seperate tab on the same screen 

![image](https://github.com/umbraco/Umbraco-CMS/assets/98405431/50746e76-3207-4cc8-9f16-a398b80e9b3b)

### Specifics

This affects all browsers, there are no errors in the event logs relating to this.

### Steps to reproduce

STEPS
1. Load the url for Umbraco v 10.6.0 or v12.2.0
2. Login with valid credentials
3. Once logged in, run at lighthouse report for Accessibility on Desktop on the 'Prod Sync' tab
4. Repeat step 3 for the 'Get Started' tab

![image](https://github.com/umbraco/Umbraco-CMS/assets/98405431/cd268acd-43a6-4fe5-98b9-38804bba2e58)
![image](https://github.com/umbraco/Umbraco-CMS/assets/98405431/27f66c8c-149a-447f-839f-641712ba6e43)

### Expected result / actual result

Lighthouse Accessibility Audit does not return any issues

**Comment by andr317c** [comment]
Hey! Thanks for reporting this issue. I was able to reproduce these accessibility errors on versions 10.7.0 and 12.2.0 for the 'Get Started' tab. 
![image](https://github.com/umbraco/Umbraco-CMS/assets/73799582/43564a95-2973-4bb6-b8c4-bec3f9b409cd)
I was unable to find the 'Sync Prod' tab in Umbraco. 
 
 This issue is something we would like help with, I'll set this issue as up for grabs 😄

**Comment by nielslyngsoe** [comment]
I'm closing this issue as the affected features and versions are no longer present in the Umbraco CMS. (V.13 is in the security phase and not receiving new updates unless it is regarding security)

This is fixed in newer versions of the CMS, so I would advise upgrading to a later version.

Thanks for taking the time to report it in the first place — we do appreciate it, even when we can't get every issue over the finish line.
